.github/workflows/ci.yaml

```diff
@@ -109,6 +109,7 @@ jobs:
       - name: Run formatting check
         run: |
           . venv/bin/activate
+          pip install -r requirements_test_pre_commit.txt # Pylint is a system hook we need it
```

Comment:
I haven't looked at the action logs, but we do restore the Python venv here which includes `pylint` already.

Reply:
I removed that